concent_api/core/tests/test_integration_report_computed_task.py

```diff
@@ -2612,3 +2611,168 @@ def test_provider_forces_computed_task_report_and_requestor_sends_rejection_due_
         self._assert_stored_message_counter_not_increased()
 
         self._assert_client_count_is_equal(2)
+
+    def test_that_message_report_computed_task_in_database_is_replaced_when_requestor_send_diffrent_report_computed_task(self):
+        # Expected message exchange:
+        # Provider  -> Concent:    MessageForceReportComputedTask
+        # Concent   -> Requestor:  MessageForceReportComputedTask
+        # Requestor -> Concent:    MessageAckReportComputedTask (diffrent ReportComptedTask)
+        # Concent   -> Provider:   MessageAckReportComputedTask
+
+        # STEP 1: Provider forces computed task report via Concent
+
+        task_to_compute = self._get_deserialized_task_to_compute(
+            timestamp = "2017-12-01 10:00:00",
+            task_id = '1',
+            subtask_id = '8',
+            deadline = "2017-12-01 11:00:00"
+        )
+
+        report_computed_task_for_requestor = self._get_deserialized_report_computed_task(
+            timestamp = "2017-12-01 10:58:00",
+            task_to_compute = task_to_compute,
+        )
+        report_computed_task_for_concent = self._get_deserialized_report_computed_task(
```

Comment:
and this one (if I understand correctly) is `report_computed_task_from_provider`

Reply:
Done